api/common/v1alpha1/jsonpatch.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"strconv"
 
-	"github.com/evanphx/json-patch/v5"
+	jsonpatch "github.com/evanphx/json-patch/v5"
```

Comment:
Feels like it's left over
```suggestion
                    "github.com/evanphx/json-patch/v5"
```